### PR TITLE
Poster introduction

### PR DESCRIPTION
**Pull request by joristt**
Wrote very short introduction. It's not that I'm lazy, but i genuinely don't think that it needs to be longer. If you miss anything, tell me.

### CODE REVIEW

**Comment by joristt** [comment]
Whoops, this PR also contains everything Orestis wrote. I'll review his PR first.

File: poster/poster.tex [hunk 1]
```diff
@@ -70,7 +70,7 @@
             
 \begin{block}{Introduction}
 	\begin{itemize}
-		\item Janus is an imperative reversible programming language, meaning that every computation and function can be reversed. \textbf{Hanus} is an extended implementation of Janus that can be compiled straight to Haskell. Because of this, Hanus contains many awesome Haskell features!
```
Comment (omelkonian) [inline poster/poster.tex]: Maybe split it to a second `\item` in `\textbf{Hanus}`?

Comment (joristt) [reply poster/poster.tex]: Agreed.

**Comment by omelkonian** [comment]
Gut, i will merge it when you merge mine :D

**Comment by omelkonian** [comment]
Rebase plz

**Comment by omelkonian** [comment]
I presume something went wrong with the rebase, since I still see my merged commits on this PR.

**Comment by joristt** [comment]
Whoops, yeah I've screwed up terribly here, let me see if I can still fix this :P.

**Comment by omelkonian** [comment]
nvm, i will rebase it

**Pull request merged**